app.js

```diff
@@ -0,0 +1,54 @@
+const hamBurger = document.querySelector('.hambuger');
+
+function clicked () {
+  let div = document.createElement('div');
+  div.className = 'modal';
+  let img = document.createElement('img');
+  img.className = 'icon';
+  img.src = './images/close.png';
```

Comment:
- [ ] Great job with the mobile menu. I recommend fixing the `x`(close) button. Its not being displayed. I think changing `./images/...` to `./Images/...` will fix the issue 
**imp**
![image](https://user-images.githubusercontent.com/37901469/167474619-84f3e3f1-e7b6-4dc1-8777-633fc97cc9b9.png)